package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "gatsby-starter-hello-world",
+  "private": true,
+  "description": "A simplified bare-bones starter for Gatsby",
+  "version": "0.1.0",
+  "license": "MIT",
+  "scripts": {
+    "build": "gatsby build",
+    "develop": "gatsby develop",
+    "format": "prettier --write \"**/*.{js,jsx,json,md}\"",
```

Comment:
Looks like gatsby sets you up with prettier out of the box. I'd recommend getting into the habit of running the format command before committing code. Or if you prefer, you can configure your editor to format "on save" which is my preference but not everyone shares that opinion. I find it easier to forget if it isn't happening automatically.

Beyond that I'd recommend checking out and setting up ESLint. It may be a little overkill for a smaller personal project of this size but it's an important tool to know how to use and getting some experience setting it up would be valuable.